benchexec/tablegenerator/react-table/src/utils/utils.js

```diff
@@ -20,6 +20,7 @@ const prepareTableData = ({ head, tools, rows, stats, props }) => {
         ...column,
         colIdx: idx,
       })),
+      scoreBased: rows[0].results[idx].score !== undefined,
```

Comment:
Doesn't this just check the first row? Could happen that we do not have a score there but in other rows.

Reply:
Yes, I was assuming that given a score exists, it will be defined in all rows. I was trying to avoid to use `Array.some(...)` because in case there are no scores (which will be most of the time currently), all rows will unnecessarily be iterated through all the time. I guess there's no way around it then, right?

Reply:
Yes.

Will this be checked each time a plot is opened? Then we could cache it.

Reply:
No, this is only done one time on the initial render of the `Overview` component.
I've applied the change.